esp-wifi/src/ble/os_adapter_esp32s3.rs

```diff
@@ -127,17 +135,67 @@ pub(super) static G_OSI_FUNCS: osi_funcs_s = osi_funcs_s {
     coex_core_ble_conn_dyn_prio_get: Some(
         ble_os_adapter_chip_specific::coex_core_ble_conn_dyn_prio_get,
     ),
+    coex_schm_register_btdm_callback: Some(coex_schm_register_btdm_callback),
     coex_schm_status_bit_set: Some(coex_schm_status_bit_set),
     coex_schm_status_bit_clear: Some(coex_schm_status_bit_clear),
+    coex_schm_interval_get: Some(coex_schm_interval_get),
+    coex_schm_curr_period_get: Some(coex_schm_curr_period_get),
+    coex_schm_curr_phase_get: Some(coex_schm_curr_phase_get),
     interrupt_on: Some(ble_os_adapter_chip_specific::interrupt_on),
     interrupt_off: Some(ble_os_adapter_chip_specific::interrupt_off),
     esp_hw_power_down: Some(ble_os_adapter_chip_specific::esp_hw_power_down),
     esp_hw_power_up: Some(ble_os_adapter_chip_specific::esp_hw_power_up),
     ets_backup_dma_copy: Some(ble_os_adapter_chip_specific::ets_backup_dma_copy),
     ets_delay_us: Some(ets_delay_us_wrapper),
     btdm_rom_table_ready: Some(btdm_rom_table_ready_wrapper),
+    coex_bt_wakeup_request: Some(coex_bt_wakeup_request),
+    coex_bt_wakeup_request_end: Some(coex_bt_wakeup_request_end),
 };
 
+extern "C" fn coex_schm_register_btdm_callback(_callback: *const ()) -> i32 {
+    trace!("coex_schm_register_btdm_callback");
```

Comment:
Is this ever called? If not, should this be `unimplemented!()` instead, so that we don't do something wrong?

Reply:
Thanks - I forgot to implement it